chunking/semantic-chunking.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Semantic Chunking
+---
+
+Semantic chunking is a method of splitting documents into smaller chunks by analyzing semantic similarity between text segments using embeddings. It uses the chonkie library to identify natural breakpoints where the semantic meaning changes significantly, based on a configurable similarity threshold. This helps preserve context and meaning better than fixed-size chunking by ensuring semantically related content stays together in the same chunk, while splitting occurs at meaningful topic transitions.
+
+```python cookbook/chunking/semantic_chunking.py
```

Comment:
```suggestion
```python
```

Reply:
We keep the file name. It's consistent throughout the docs. Also points to the cookbook in this case

Reply:
Other docs don't have any doc name or just `agent.py`

Reply:
E.g. https://docs.phidata.com/knowledge/document

Reply:
I don't mind that much, as long as we are consistent